setup.cfg

```diff
@@ -0,0 +1,16 @@
+[flake8]
+max-line-length=99
```

Comment:
Why not 100? Much easier to remember.

Reply:
I based this off PEP8 having a 79 character limit for an 80 character target. It seems that some editors start line wrapping one character earlier. See https://www.reddit.com/r/learnpython/comments/1h2eug/pep_8_why_is_the_character_limit_79_and_not_80/

However, I think it'd be fine to set this to 100 as well.